src/main/java/com/kodilla/ecommercee/domain/Product.java

```diff
@@ -34,7 +40,16 @@ public String getDescription() {
         return description;
     }
 
-    public double getPrice() {
+    public int getPrice() {
```

Comment:
Cena nie może być z groszami?

Reply:
W JSONach nie ma groszy. Nie wiem w sumie czy to uzasadniona modyfikacja.

Reply:
Ja bym się z niej wycofał do czasu zweryfikowania we frontendzie co on wyświetla i jak.